### PR TITLE
Re-add mgmt certs

### PR DESCRIPTION
**Pull request by jacob-delgado**
## Description

Re-open https://github.com/istio/istio.io/pull/13344

## Reviewers

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+Only P256 and P384 are supported via `ECC_CURVE`.
+
+If you prefer to retain RSA signature algorithms and want to modify the RSA key size,
+you can change the value of `WORKLOAD_RSA_KEY_SIZE`.
+
```
Comment (windsonsea) [inline content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md:167]: line 167 can be removed to pass the lint check:

```
./en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md:167:
MD012 Multiple consecutive blank lines
```

File: content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+weight: 30
+keywords: [traffic-management,proxy]
+owner: istio/wg-networking-maintainers,istio/wg-environments-maintainers
+test: n/a
+---
+
+{{< boilerplate experimental >}}
+
+Many users need to manage the types of the certificates used within their environment. For example,
+some users require the use of Elliptical Curve Cryptography (ECC) while others may need need a
```
Comment (windsonsea) [inline content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md:14]:
```suggestion
some users require the use of Elliptical Curve Cryptography (ECC) while others may need to use a
```

**Review by windsonsea — COMMENTED** [review]
find a typo

File: content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+      ...
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+### Signature Algorithm
+
+By default, the sidecars will create RSA-based certificates. If you want to change it to
```
Comment (dhawton) [inline content/en/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md:131]:
```suggestion
By default, the sidecars will create RSA certificates. If you want to change it to
```

File: .spelling [hunk 4]
```diff
@@ -952,6 +952,7 @@ rollouts
 routable
 RPC
 RPCs
+RSA-based
```
Comment (dhawton) [inline .spelling:955]:
```suggestion
```

**Review by dhawton — APPROVED** [review]

**Pull request merged**